lambda_functions/v2/functions/csv-forwarder/main.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type Lambda struct {
```

Comment:
This is going to be a pain and it is a little nitpick but the effective Go handbook states
> All top-level, exported names should have doc comments, as should non-trivial unexported type or function declarations.

The main reason for this is Go has the `go doc` command line tool which will automatically generate code docs for us and it will inform users what certain exported types/functions do if they have the comments to go with it

More information here: https://go.dev/doc/comment